paris_rollins.py

```diff
@@ -25,21 +35,39 @@
 WORKER_NICE = 19
 # paris-traceroute should take no longer than this to complete (timed out,
 # partial results will be discarded).
-WORKER_TIMEOUT = 20 
+WORKER_TIMEOUT = 60 
 # Maximum number of paris-traceoutes to run simultaneously (requests to run
 # more will be discarded).
 MAX_WORKERS = 10
+# Whether a TCP connection is closed.
+WEB100_STATE_CLOSED = 1
+# Whether a TCP connection was over IPv4
+WEB100_IPV4 = 1
+# Base source port to use when running traceroute
+PARIS_TRACEROUTE_SOURCE_PORT_BASE = 33457
+# Where to log traceroutes
+LOG_PATH = '/tmp'
```

Comment:
Because it's for experimental testing - don't care that the results are discarded, in fact want them in a place that definitely won't be collected by the pipeline.

I added CLI flag nonetheless to change it.